index.d.ts

```diff
@@ -62,6 +62,11 @@ interface NotToggleCustomControlProps extends CommonCustomControlProps {
   handleOnChange?(checked: boolean): void;
 }
 
+interface RuleGroupCustomControlProps extends CommonCustomControlProps {
+  checked: boolean
+  handleOnChange?(checked: boolean): void;
+}
+
```

Comment:
These are incorrect.  They should match the props for `<RuleGroup />`.  Something like this:

```ts
interface RuleGroupCustomControlProps {
  id?: string;
  parentId?: string;
  combinator?: string;
  rules?: [];
  translations?: {};
  schema?: {};
  not?: boolean;
}
```
We may need to specify the `schema` attributes.  For `translations`, we can reuse the definition from below under `interface QueryBuilderProps {...}`.